base/testfiles-lthooks2/shipout2-010.tlg

```diff
@@ -119,9 +117,9 @@ Absolute page = 4 (target: 4)
 Completed box being shipped out [2]
 \vbox(633.0+0.0)x407.0
 .\hbox(0.0+0.0)x0.0
-..\kern -72.27
-..\vbox(0.0+0.0)x0.0, glue set 72.27fil
-...\kern -72.27
+..\kern -72.26999
+..\vbox(0.0+0.0)x0.0, glue set 72.26999fil
+...\kern -72.26999
```

Comment:
is that a calculated 1in? interesting that it comes out differently in the two approaches (not that it matters)